js2-mode.el

```diff
@@ -10499,7 +10525,7 @@ and expression closure style is also supported
 
 POS is the start position of the `get' or `set' keyword.
 PROP is the `js2-name-node' representing the property name.
-GET-P is non-nil if the keyword was `get'."
+TYPE-STRING is a string `get', `set', or `', indicating a found keyword."
```

Comment:
Was the last option meant to be an asterisk? I think nil is also a possible value.